CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ option( ALGEBRA_PLUGINS_INCLUDE_VECMEM
    "Include VecMem types in Algebra Plugins" FALSE )
 option( ALGEBRA_PLUGINS_BUILD_TESTING "Build the unit tests of Algebra Plugins"
    TRUE )
+option( ALGEBRA_PLUGINS_BUILD_BENCHMARKS "Build the benchmark suite of Algebra Plugins"
```

Comment:
I don't think we need this for now because `TRUE` option will make an error. It will be okay to add this option once you have something in `benchmarks` directory.

Reply:
I plan on adding some tests in the next PR itself. Plus, the default option (for now, at least), is `FALSE`. However, if you think it will cause problems (maybe in the CI or something), then I can remove it and add it into the next PR.

Reply:
You can leave it if you like